### PR TITLE
Implementation of phylogenetic tree

### PR DESCRIPTION
**Pull request by tadzjibov**

### CODE REVIEW

File: src/main/java/io/github/programminglife2016/pl1_2016/parser/PhyloGeneticTreeParser.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
+package io.github.programminglife2016.pl1_2016.parser;
+
+import io.github.programminglife2016.pl1_2016.Launcher;
```
Comment (smesseim) [inline src/main/java/io/github/programminglife2016/pl1_2016/parser/PhyloGeneticTreeParser.java]: This import is unnecessary.

File: src/main/java/io/github/programminglife2016/pl1_2016/parser/PhyloGeneticTreeParser.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        current.setChildren(nodes);
+        return current;
+    }
+
+
+    /**
+     * Test parser.
+     * @param args args.
+     */
+    public static void main(String[] args) {
```
Comment (smesseim) [inline src/main/java/io/github/programminglife2016/pl1_2016/parser/PhyloGeneticTreeParser.java]: Remove this main method. Testing should be done in an actual JUnit test.

File: src/main/java/io/github/programminglife2016/pl1_2016/parser/TreeNode.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
+package io.github.programminglife2016.pl1_2016.parser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class for representing a node in the phylogenetic tree.
+ */
+public class TreeNode implements JsonSerializable {
```
Comment (smesseim) [inline src/main/java/io/github/programminglife2016/pl1_2016/parser/TreeNode.java]: Wouldn't it be better if `TreeNode` was an interface? `JsonSerializable` is not specific as an interface.

File: src/main/java/io/github/programminglife2016/pl1_2016/parser/TreeNode.java [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+     * Convert tree to string.
+     * @param c level indent.
+     */
+    public void toString(String c) {
+        System.out.print(c + "#" + this.getId() + " " + getWeight());
+        if (parent != null) {
+           System.out.println(" " + parent.getId());
+        } else {
+           System.out.println();
+        }
```
Comment (smesseim) [inline src/main/java/io/github/programminglife2016/pl1_2016/parser/TreeNode.java]: Stop polluting stdout.

**Comment by smesseim** [comment]
Multiple python programs are checked in. How often will they be used? If they're only used once per dataset, I don't think they belong in this repository.

**Comment by RaviShivam** [comment]
The python scripts are only used once to parse the metadata in excel format to csv format, I have therefor removed them.

**Comment by smesseim** [comment]
LGTM

**Comment by SamuelSital** [comment]
LGTM

**Pull request merged**